src/term-spec.js

```diff
@@ -35,6 +35,10 @@ declare export class MethodDefinition extends NamedObjectProperty {
   body: FunctionBody;
 }
 
+declare export class VariableReference extends Term {
+  name: any; // Identifier (string)
```

Comment:
The current spec macro uses `any` for anything that is not a Term (anything the reducer doesn't recurse into). Not ideal so I'm also working on rewriting the spec macro (with parser combinators!) but that's taking a while.